api/tests/TransformerTest.php

```diff
@@ -96,23 +102,28 @@ public function testRelationSelf()
         }
     }
 
-//    public function testPaginated()
-//    {
-//        $entities = factory(App\Models\TestEntity::class, 3)->make();
-//
-//        $paginated = new LengthAwarePaginator($entities, 3, 1, 1);
-//
-//        $collection = $this->service->paginatedCollection($paginated);
-//
-//        $this->assertTrue(is_array($collection));
-//    }
-//
-//    public function testParseIncludes()
-//    {
-//        $entity = factory(App\Models\TestEntity::class)->make();
-//
-//        $this->service->parseIncludes('foobar');
-//
-//        $item = $this->service->item($entity);
-//    }
+    public function testPaginated()
+    {
+
+        $total = 30;
+        $perPage = 5;
+
+        $entities = factory(App\Models\TestEntity::class, $total)->make();
+
+        $paginated = new LengthAwarePaginator($entities, $total, $perPage, 1);
+
+        $collection = $this->service->paginatedCollection($paginated);
+
+        $this->assertTrue(is_array($collection['data']));
+        $this->assertCount($perPage, $collection['data']);
+    }
+
+    public function testParseIncludes()
+    {
+        $entity = factory(App\Models\TestEntity::class)->make();
+
+        $this->service->parseIncludes('foobar');
+
+        $item = $this->service->item($entity);
```

Comment:
I've just returned it as it was before =)))
Do we really need parse includes anyway?